docs/buildah-from.md

```diff
@@ -47,13 +41,13 @@ If an image needs to be pulled from the registry, suppress progress output.
 
 ## EXAMPLE
 
-buildah from imagename --pull --transport "docker://myregistry.example.com/"
+buildah from imagename --pull
 
 buildah from docker://myregistry.example.com/imagename --pull
 
 buildah from imagename --signature-policy /etc/containers/policy.json
 
-buildah from imagename --pull-always --transport "docker://myregistry.example.com/" --name "mycontainer"
+buildah from docker://myregistry.example.com/imagename --pull-always --name "mycontainer"
```

Comment:
That man page rocks! I think you should mention the push man page from man pages which operate on images.